test/addons/rbd-mirror/test-data/kustomization.yaml

```diff
@@ -3,7 +3,7 @@
 
 ---
 resources:
-- vr-sample.yaml
+- vr.yaml
```

Comment:
We should rename vr.yaml to vr-1m.yaml to make it easy to add another vr (e.g. vr-5m.yaml) later and have consistent names. Same for the vgr.